src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -254,7 +254,7 @@ private void init() {
 
             // groupSidePane.getToggleCommand().setSelected(sidePaneManager.isComponentVisible(GroupSidePane.class));
```

Comment:
Can this be removed or rewritten?

Reply:
At some point we should show the toggle state in the menu and I would hence leave this there as a reminder. Sadly, there is still a lot of such uncommented old in the maintable / frame / base panel classes...